src/core/views/widget.js

```diff
@@ -227,13 +226,20 @@ class Widget extends Component {
       iframeRef.contentWindow != null
 
     if (canPostMessage) {
-      iframeRef.contentWindow.postMessage('embed-focus', '*')
-      this.setState({
-        isIframeFocused: true
-      })
+      this.setState({ isIframeFocused: true }, this.sendFocusMessageToIframe)
     }
   }
 
+  sendFocusMessageToIframe () {
+    const { iframeRef } = this.iframe
+    if (!iframeRef) { return }
+
+    setTimeout(
```

Comment:
`setTimeout` is the only change, right?

Reply:
There're two, actually. It was quite hard to tackle, but the issue was because the event was not always triggered after the state is set. I'm using two things: a) setState callback, b) setTimeout

Reply:
it's issue no 353

Reply:
Please do not forget to include that in commit message when you squash :) Thanks.